tests/test_unary_pointwise_ops.py

```diff
@@ -780,6 +780,20 @@ def test_accuracy_logical_not(shape, dtype):
     gems_assert_equal(res_out, ref_out)
 
 
+@pytest.mark.log
+@pytest.mark.parametrize("shape", POINTWISE_SHAPES)
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+def test_accuracy_log(shape, dtype):
+    inp = torch.rand(shape, dtype=dtype, device=flag_gems.device)
+
+    ref_inp = to_reference(inp)
+    ref_out = torch.log(ref_inp)
+    with flag_gems.use_gems():
+        res_out = torch.log(inp)
+
+    gems_assert_equal(res_out, ref_out)
```

Comment:
as for vendors need to keep aligned with cpu reference, I suggest upcasting the reference input to fp64 and asserting close for outputs.